docs/en/compute/AWS-EMR.md

```diff
@@ -0,0 +1,123 @@
+to get the 'Master public DNS'. SSH into this instance using the keypair provided in the previous command.
+4. Test that Alluxio is running as expected by running `sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio runTests"`
+
+Alluxio is installed in `/opt/alluxio/` by default. Hive and Presto are already configured to connect to Alluxio. The
+cluster also uses AWS Glue as the default metastore for both Presto and Hive. This will allow you to maintain table
+definitions between multiple runs of the Alluxio cluster.
+
+## Creating a Table
+
+The simplest step to using EMR with Alluxio is to create a table on Alluxio and query it using Presto/Hive.
+
+1. SSH into the 'hadoop' user in the master node. Then switch to the 'alluxio' user.
+2. Create a directory in Alluxio to be the external location of your table.
+```bash
+/opt/alluxio/bin/alluxio fs mkdir /testTable
+```
+3. Set the 'hadoop' user to be the owner of the directory
+```bash
+/opt/alluxio/bin/alluxio fs chown hadoop:hadoop /testTable
+```
+4. Exit to switch back into the 'hadoop' user and start the hive CLI.
+```bash
+hive
+```
+5. Create a new database to see if AWS Glue is working as expected. Check the [console](https://console.aws.amazon.com/glue/home)
+to see if the database is created.
+```sql
+CREATE DATABASE glue;
+```
+6. Use the newly created database and define a table.
+```sql
+USE glue;
+create external table test1 (userid INT,
+age INT,
+gender CHAR(1),
+occupation STRING,
+zipcode STRING)
+ROW FORMAT DELIMITED
+FIELDS TERMINATED BY '|'
+LOCATION 'alluxio:///testTable';
+```
+7. Create the Presto /tmp directory
+```bash
+#Create Presto temp directory
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio fs mkdir /tmp"
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio fs chmod 777 /tmp"
+presto-cli --catalog hive
+```
+8. Insert values into the table
+```sql
+USE glue;
+INSERT INTO test1 VALUES ('1', 24, 'F', 'Developer', '12345');
+```
+9. Read back the values in the table
+```sql
+SELECT * FROM test1;
+```
+
+##Customization
+Tuning of Alluxio properties can be done in a few different locations. Depending on which service needs tuning, EMR
+offers different ways of modifying the service settings/environment variables.
+
+###Alluxio Service
+Any server-side configuration changes must be made in the `alluxio-emr.sh` bootstrap script. In the section for generating
+the `alluxio-site.properties`, add a line with the configuration needed to append to the bottom of the file. Options can also
+be passed as the 3rd argument to the bootstrap script with a ';' delimiter.
+
+###Alluxio Client
+Generic client-side properties can also be edited via the bootstrap script as mentioned above. This is mostly for the native
+client (CLI). Property changes for a specific service like Presto/Hive should be done in the respective configuration file
+i.e. core-site.xml, hive.catalog.
```

Comment:
quote `` `core-site.xml` `` and `` `hive.catalog` ``?

Reply:
Done